app/models/saved_claim.rb

```diff
@@ -173,13 +173,21 @@ def validate_form(schema)
   # This method exists to change the json_schemer errors
   # to be formatted like json_schema errors, which keeps
   # the error logging smooth and identical for both options.
+  # This method also filters out the `data` key because it could
+  # potentially contain pii.
   def reformatted_schemer_errors(errors)
-    errors.map!(&:symbolize_keys)
-    errors.each do |error|
-      error[:fragment] = error[:data_pointer]
-      error[:message] = error[:error]
+    errors.map do |error|
+      symbolized = error.symbolize_keys
+      {
+        data_pointer: symbolized[:data_pointer],
+        error: symbolized[:error],
+        details: symbolized[:details],
+        schema: symbolized[:schema],
+        root_schema: symbolized[:root_schema],
+        message: symbolized[:error],
+        fragment: symbolized[:data_pointer]
```

Comment:
Are these right? should it be this instead?
```suggestion
        message: symbolized[:message],
        fragment: symbolized[:fragment]
```

Reply:
Yeah, those are correct. This was done previously to keep the naming conventions for errors from the old gem that was used for validation. I put them both at the bottom to try to make it more clear they are the ones that are different.